### PR TITLE
Release v0.23.12

### PR DESCRIPTION
**Pull request by rossabaker**

### CODE REVIEW

**Comment by rossabaker** [comment]
If anyone has unmerged issues they really want in this release, please milestone them or mention them here.  I would like to crank through this today, if possible.

File: docs/changelog.md
Comment (armanbilge) [inline docs/changelog.md:31]: Cool, so in the end we will not be keeping these in sync? 🙏

Comment (rossabaker) [reply docs/changelog.md:31]: Certainly not for 1.0.  We could for 0.23, but a handful would already benefit from a 0.24 anyway (servlet ecosystem, scalatags).  I'm thinking for the rest, maybe we see if users freak out about there not being an 0.23.12, and what maintainers step up?

Comment (armanbilge) [reply docs/changelog.md:31]: > I'm thinking for the rest, maybe we see if users freak out about there not being an 0.23.12, and what maintainers step up?

💯 this is exactly what I've been thinking.

**Comment by danicheg** [comment]
A few PRs are waiting for review for a relatively long-time and targeting the 0.23 series. I'd be happy if we consider them for inclusion in this release: https://github.com/http4s/http4s/pull/6204 https://github.com/http4s/http4s/pull/6316

**Comment by armanbilge** [comment]
Oh, shoot, I almost forgot, I thought more and I decided we should keep all the JS-only APIs private. PR forthcoming.

**Comment by armanbilge** [comment]
Btw, since these release notes are describing the schism so nicely, seems a shame not to include blaze on that list as well. We're only blocked by https://github.com/http4s/http4s/pull/6204 and https://github.com/http4s/http4s/pull/6387 and then we can schism it as well.

**Comment by ChristopherDavenport** [comment]
The retryKey, but I don't see any of the other changes mentioned. So I figure that is still incoming?

File: docs/changelog.md
Comment (armanbilge) [reply docs/changelog.md]: Yup, also thinking the same when I saw that, but didn't want to jinx it 😕

File: docs/changelog.md
Comment (armanbilge) [inline docs/changelog.md:40]: We didn't end up doing this, right?

File: docs/changelog.md
Comment (armanbilge) [inline docs/changelog.md:50]: This goes in another section.

**Comment by rossabaker** [comment]
CI ran on [3966296](https://github.com/http4s/http4s/pull/6403/commits/39662966f8e5b679c57d7dd69afca9cd94b743f2) to prove that Laika can digest the duplicate headers.  I skipped it on the subsequent tweaks for expediency.

**Review by armanbilge — APPROVED** [review]

**Review by ChristopherDavenport — APPROVED** [review]

**Pull request merged**
